src/components/account/IFXAccountList.vue

```diff
@@ -1,55 +1,74 @@
 <script>
-import IFXActionSelect from '@/components/action/IFXActionSelect'
-import IFXSearchField from '@/components/IFXSearchField'
+import IFXAccountMixin from '@/components/account/IFXAccountMixin'
 import IFXItemDataTable from '@/components/item/IFXItemDataTable'
+import IFXSearchField from '@/components/IFXSearchField'
 import IFXItemListMixin from '@/components/item/IFXItemListMixin'
-import accountMixin from '@/components/account/IFXAccountMixin'
 
 export default {
-  name: 'IFXAccountList',
-  mixins: [accountMixin, IFXItemListMixin],
+  // TODO: fix IFXAccountList in ifxvue and then use that
```

Comment:
I'm not sure what this comment refers to since this *is* IFXAccountList in IFXVue, right?